plugins/OnlineSales.Plugin.SendGrid/Migrations/20230404175756_SendGridEvent table renaming.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
@IgorKostyukov please collapse all changes into a single migration before it gets merged into the develop branch.